testsuite/smoke-test/src/keyless.rs

```diff
@@ -254,25 +298,77 @@ async fn get_transaction(
     LocalSwarm,
     SignedTransaction,
 ) {
-    let (tw_sk, config, jwk, mut swarm) = setup_local_net().await;
+    let (tw_sk, config, jwk, mut swarm, _) = setup_local_net().await;
 
     let (sig, pk) = get_pk_and_sig_func();
 
     let mut info = swarm.aptos_public_info();
-    let signed_txn =
-        sign_transaction(&mut info, sig.clone(), pk.clone(), &jwk, &config, &tw_sk).await;
+    let signed_txn = sign_transaction(
+        &mut info,
+        sig.clone(),
+        pk.clone(),
+        &jwk,
+        &config,
+        Some(&tw_sk),
+    )
+    .await;
 
     (sig, pk, swarm, signed_txn)
 }
 
-async fn setup_local_net() -> (Ed25519PrivateKey, Configuration, RSA_JWK, LocalSwarm) {
+async fn setup_local_net() -> (
+    Ed25519PrivateKey,
+    Configuration,
+    RSA_JWK,
+    LocalSwarm,
+    CliTestFramework,
+) {
     let (mut swarm, mut cli, _faucet) = SwarmBuilder::new_local(1)
         .with_aptos()
         .build_with_cli(0)
         .await;
 
     let (tw_sk, config, jwk) = spawn_network_and_execute_gov_proposals(&mut swarm, &mut cli).await;
-    (tw_sk, config, jwk, swarm)
+    (tw_sk, config, jwk, swarm, cli)
+}
+
+async fn remove_training_wheels(swarm: &mut LocalSwarm, cli: &mut CliTestFramework) {
+    let client = swarm.validators().next().unwrap().rest_client();
+    let root_idx = cli.add_account_with_address_to_cli(
+        swarm.root_key(),
+        swarm.chain_info().root_account().address(),
+    );
+    let jwk_patch_script = format!(
+        r#"
+script {{
+use aptos_framework::{};
+use aptos_framework::aptos_governance;
+use std::option;
+fun main(core_resources: &signer) {{
+    let framework_signer = aptos_governance::get_signer_testnet_only(core_resources, @0000000000000000000000000000000000000000000000000000000000000001);
+    {}::update_training_wheels(&framework_signer, option::none());
+}}
+}}
+"#,
+        KEYLESS_ACCOUNT_MODULE_NAME, KEYLESS_ACCOUNT_MODULE_NAME
+    );
+    let txn_summary = cli.run_script(root_idx, &jwk_patch_script).await.unwrap();
+    debug!("txn_summary={:?}", txn_summary);
+
+    let mut info = swarm.aptos_public_info();
+
+    // Increment sequence number since we patched a JWK
```

Comment:
Do you wanna update this comment? There is no JWK update...

Reply:
Done